packages/wonder-blocks-popover/components/popover-context.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import * as React from "react";
+
+import type {Placement} from "@khanacademy/wonder-blocks-tooltip";
+
+type ContextType = {|
+    close?: () => mixed,
+    placement?: Placement,
+|};
+
+const defaultContext: ContextType = {
+    close: undefined,
+    placement: "top",
```

Comment:
Added `placement` to the context to facilitate passing this value from `Popover` (via `TooltipPopper`) down to `PopoverContent`. This is needed here to reposition the illustration to the start or the end of the content, in case the popper changes its `placement`.